airflow-core/src/airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -334,6 +336,7 @@ def get_dag_runs(
     readable_dag_runs_filter: ReadableDagRunsFilterDep,
     session: SessionDep,
     dag_bag: DagBagDep,
+    run_id_pattern: Annotated[_SearchParam, Depends(search_param_factory(DagRun.run_id, "run_id"))],
```

Comment:
```suggestion
    run_id_pattern: Annotated[_SearchParam, Depends(search_param_factory(DagRun.run_id, "run_id_pattern"))],
```

(All search params end up with the `pattern` suffix, this will be the name of the query parameter, I missed that in the previous review)

Reply:
I updated `run_id` to `run_id_pattern` to keep the postfix just like you suggested: https://github.com/apache/airflow/pull/52437/commits/8c3ef8562fcb8f816724477a339072257589e318
Thanks for spotting this!